sickbeard/providers/scenetime.py

```diff
@@ -87,7 +85,7 @@ def search(self, search_strings, age=0, ep_obj=None):  # pylint: disable=too-man
                     logger.log('Search string: {search}'.format
                                (search=search_string), logger.DEBUG)
 
-                search_url = self.urls['search'] % (quote(search_string), self.categories)
+                search_url = self.urls['search'].format(search_string, self.categories)
```

Comment:
This has been tested and it is working as expected. It was supposed to be a quick fix for the provider, as it didn't work at all. If you can wait until tomorrow, I can change the strings to params, although the provider would need a full rewrite.